src/app/(website)/layout.tsx

```diff
@@ -1,32 +1,25 @@
-import React from 'react';
-
 import { Stack } from '@mantine/core';
-import { Roboto } from 'next/font/google';
-
 import { Footer } from '@/entities/website/components/Footer';
-import { Header } from '@/entities/website/components/Header';
-
-const roboto = Roboto({
-    variable: '--font-roboto',
-    subsets: ['latin', 'cyrillic'],
-});
+import { Header } from '@/features/website/header';
+import React from 'react';
 
 export default function WebsiteLayout({
     children,
 }: {
     children: React.ReactNode;
 }) {
     return (
-        <div className={roboto.className}>
-            <Stack
-                mih="100dvh"
-                px={{ base: 16, sm: 20, md: 24 }}
-                py={{ base: 8, sm: 12, md: 16 }}
-            >
-                <Header />
-                <main style={{ flex: 1 }}>{children}</main>
-                <Footer />
-            </Stack>
-        </div>
+        <Stack
+            mih="100dvh"
+            px={{ base: 16, sm: 20, md: 24 }}
+            py={{ base: 8, sm: 12, md: 16 }}
+            style={{
+                background: 'var(--Gray-10, rgba(248, 249, 250, 1))',
+            }}
```

Comment:
Я до кінця ще не зрозумів як ми юзаємо стилі, але як на мене краще уникати напряму використовувати style. І ще не дуже зрозумів навіщо тут rgba формат, якщо там 1 в opacity. Можна зробити просто 
```suggestion
            bg="gray.0"
```

Reply:
Пітримую, краще використовувати кольори мантіни